### PR TITLE
refactor: refactored useCategory for better dx experience

### PR DESCRIPTION
**Pull request by Frodigo**
## Description
Refactored useCategory composable for better DX experience

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: packages/theme/composables/useCategory/index.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export const useCategory = (): UseCategory => {
   const categories: Ref<Array<Category>> = ref(null);
```
Comment (sethidden) [inline packages/theme/composables/useCategory/index.ts:15]: On line 13 above there's the "search" that should be changed to "load"

File: packages/theme/composables/useCategory/index.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ export const useCategory = (): UseCategory => {
...
 
   // eslint-disable-next-line consistent-return
-  const search = async (searchParams: CategoryListQueryVariables) => {
+  const load = async (searchParams: CategoryListQueryVariables) => {
     Logger.debug('useCategory/search', searchParams);
```
Comment (sethidden) [inline packages/theme/composables/useCategory/index.ts:19]:
```suggestion
    Logger.debug('useCategory/load', searchParams);
```

File: packages/theme/composables/useCategory/index.ts (hunk 1 above)
Comment (Frodigo) [reply packages/theme/composables/useCategory/index.ts:15]: Good catch, thanks! fixed.

**Comment by sethidden** [comment]
@VitorLuizC heads up that this affects your PR regarding useCategory documentation

**Comment by sethidden** [comment]
@VitorLuizC heads up that this affects your PR regarding useCategory documentation

**Review by sethidden — APPROVED** [review]

**Review by sethidden — APPROVED** [review]

**Pull request merged**